### PR TITLE
Introduce the Internet to the story.

### PR DESCRIPTION
**Pull request by xlgmokha**
# Why is this needed?

This is needed so that we can continue the story and introduce a vital instrument to the narrative that will be essential for the future plot line.

## What does this do?

This change introduces the character "Internet" to the story.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by MRCH33SE** [comment]
good idea but you misspelled accessible

File: README.md [hunk 1]
```diff
@@ -13,3 +13,4 @@ Then they got kidnapped at the store.
 The shopping cart had important things including; a pink tank-top, some microwavable food, and magnets (all merchandise).
 One of them started to panic and accidentally turned into a cupcake in the process.
 Neither of them knew where to look, so they both turned to the one thing that they believed that out help them find their loved one.
+The internet. The internet, of course, was only acesibale via carrier pigeon.
```
Comment (MRCH33SE) [inline README.md:16]: you misspelled accessible

Comment (MRCH33SE) [inline README.md:16]:
```suggestion
it is important to advance the plot and include new characters, but you misspelled accessible
```

Comment (code2417) [reply README.md:16]: This is a good idea. Including the internet in the story provides a new insight and perspective which can further improve the plot line.

**Review by MRCH33SE — APPROVED** [review]
good job

**Review by code2417 — APPROVED** [review]
Looks good to me.

**Pull request merged**